std/traits.d

```diff
@@ -6403,3 +6404,71 @@ unittest
     static assert(is(typeof(a) == real));
     static assert(is(typeof(b) == real));
 }
+
+/**
+ * Determine if a symbol has a given user-defined attribute.
```

Comment:
I think it would be worthwhile to add a link to the [UDA documentation](http://dlang.org/attribute.html#UserDefinedAttribute).